src/main/java/org/miracum/streams/ume/obdstofhir/mapper/mii/ResidualstatusMapper.java

```diff
@@ -0,0 +1,71 @@
+package org.miracum.streams.ume.obdstofhir.mapper.mii;
+
+import de.basisdatensatz.obds.v3.OPTyp;
+import java.util.List;
+import java.util.Objects;
+import org.apache.commons.lang3.Validate;
+import org.hl7.fhir.r4.model.CodeableConcept;
+import org.hl7.fhir.r4.model.Enumerations.ResourceType;
+import org.hl7.fhir.r4.model.Identifier;
+import org.hl7.fhir.r4.model.Observation;
+import org.hl7.fhir.r4.model.Reference;
+import org.miracum.streams.ume.obdstofhir.FhirProperties;
+import org.miracum.streams.ume.obdstofhir.mapper.ObdsToFhirMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ResidualstatusMapper extends ObdsToFhirMapper {
+
+  private static final Logger LOG = LoggerFactory.getLogger(ResidualstatusMapper.class);
+
+  public ResidualstatusMapper(FhirProperties fhirProperties) {
+    super(fhirProperties);
+  }
+
+  public Observation map(OPTyp op, Reference patient) {
```

Comment:
```suggestion
  public Observation map(OPTyp op, Reference patient, Reference condition) {
```

Analog zu https://github.com/bzkf/obds-to-fhir/pull/155/files#diff-19bc8d016f948222434927e3db9fea3f15c323943ade9788771de3395c00e4eeR26 kannst du noch die condition reference als parameter übergen und dann Observation.focus darauf setzen - Die Condition-reference baut dann der obds-to-bundle-mapper in: https://github.com/bzkf/obds-to-fhir/pull/173/files#diff-4f8b08222492b7285cffcddf0cdd63b7390d737af96125e170490c284352c6afR120-R121. Dann ist's perfekt :)

Reply:
Alles schon erledigt, allerdings mit Parameternamen `focus` ;)